test/blackbox-tests/test-cases/show/dune-aliases.t/run.t

```diff
@@ -0,0 +1,113 @@
+Testing the "dune show aliases" command. This command shows the aliases in the
+current directory. It acts similarly to ls. It will not show aliases that appear
+in subdirectories although this could be changed in the future.
+
+In an empty dune project, the following aliases are available.
+
+  $ dune show aliases
+  .:
+  all
+  default
+  fmt
+
+User defined aliases can be added to a dune file. These should be picked up by
+the command.
+
+  $ cat > dune << EOF
+  > (alias
+  >  (name foo))
+  > EOF
+
+  $ dune show aliases
+  .:
+  all
+  default
+  fmt
+  foo
+
+Aliases in subdirectories should not be picked up.
+
+  $ mkdir subdir
+  $ cat > subdir/dune << EOF
+  > (alias
+  >  (name bar))
+  > EOF
+
+  $ dune show aliases
+  .:
```

Comment:
I think I'd prefer the `dir:` not shown in the output because it makes it harder to write automation around `dune show aliases`. I'd imagine that in most cases users will want to filter it out.

Reply:
(Also, I think it's just confusing to see the directory in the output, particularly `.:` is a pretty obscure string.)

Reply:
Sure , I just copied thr behaviour of `ls` herr.

Reply:
Hmm, not sure which `ls` you mean: mine doesn't do that :) (And thanks!)

Reply:
For me `ls` only shows the headers when requesting multiple directories. I think I will use this behaviour since in the multiple case the headers have a use.